learn/tools-ides/intellij-plugin.md

```diff
@@ -95,7 +95,12 @@ After obtaining the ZIP file using either of the above approaches, follow the st
 
 ## Using the plugin
 
-For information on using the IntelliJ Ballerina plugin to write Ballerina programs, see [Using the IntelliJ Ballerina plugin](/learn/tools-ides/intellij-plugin/using-the-intellij-plugin).
+Click on the below links to find information on using the IntelliJ Ballerina plugin to write Ballerina programs.
```

Comment:
Do we need to say "Click on the .."? IMO, something like "Refer below links" is more appopriate.

Reply:
Normally, we don't say "refer" when the user has to click on the links. Therefore, changed as follows:

- The below sections include information on using the IntelliJ Ballerina plugin to write Ballerina programs.
- The below sections include information on the various capabilities that are facilitated by the IntelliJ Ballerina plugin for the development process.

Reply:
We need to change the same in the relevant pages as well.
https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md

Reply:
Changed in the below files:

- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md
- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/intellij-plugin.md
- https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/vscode-plugin.md
 - https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/vscode-plugin/graphical-editor.md
 - https://github.com/ballerina-platform/ballerina-dev-website/blob/master/learn/tools-ides/vscode-plugin/run-all-tests.md